crates/dc_bundle/src/legacy_definition/layout/grid.rs

```diff
@@ -76,3 +76,39 @@ pub enum OverflowDirection {
     VerticalScrolling,
     HorizontalAndVerticalScrolling,
 }
+
+#[derive(Copy, Clone, PartialEq, Debug, Serialize, Deserialize)]
+pub enum Overflow {
+    Visible,
+    Hidden,
+    Scroll,
+}
+
+impl Default for Overflow {
+    fn default() -> Self {
+        Self::Visible
+    }
+}
+
+#[derive(Deserialize, Serialize, Debug, Clone, PartialEq, Default, Copy)]
+#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
+pub enum LayoutSizing {
+    #[default]
+    Fixed,
+    Hug,
+    Fill,
+}
+
+#[derive(Copy, Clone, PartialEq, Debug, Serialize, Deserialize)]
+pub enum Display {
+    #[serde(rename = "flex")]
```

Comment:
Are these needed? I thought the `#serde(rename="xyz")` tag was to tell serde how to deserialize something, but this isn't used for deserialization I don't think.

Reply:
I don't know one way or another, but either way, they'll go away once we replace the types with the protobuf versions

Reply:
Also not sure if some of these belong in `grid`. `Display` I think can be for any layout type, and others like `LayoutSizing` are full autolayout.

Reply:
Yeah... it's fuzzy. Like I had the Display proto file that only had Display in it. I can move Display to `node_style`. What else should move?

Reply:
LayoutSizing, Overflow, OverflowDirection, and ItemSpacing should go to layout/layout_style or layout/positioning.

Reply:
I ran out of time to make these changes so I'll pick this up again when I'm back.

Reply:
ok, I'm fine with that being in a follow up pull request.

Reply:
Changes now made